pkg/operator/ceph/cluster/mon/health.go

```diff
@@ -350,6 +350,36 @@ func (c *Cluster) checkHealth(ctx context.Context) error {
 		}
 	}
 
+	if c.spec.Network.MultiClusterService.Enabled {
+		for _, mon := range c.ClusterInfo.Monitors {
+			var failMon bool
```

Comment:
nit: IMO it's nice to initialize bools to false even though it's the same
```suggestion
			failMon := false
```

Reply:
not using `failMon` bool variable anymore since we only have one condition to check now.